sda-pipeline/internal/broker/broker_test.go

```diff
@@ -1,406 +0,0 @@
-package broker
-
-import (
```

Comment:
I don't see why we don't want these (broker) tests anymore, can we get a short explanation?

Reply:
Part of the merge preparation, shouldn't be in this PR.